.vscode/extensions.json

```diff
@@ -6,7 +6,8 @@
     "recommendations": [
         "octref.vetur",
         "stylelint.vscode-stylelint",
-        "esbenp.prettier-vscode"
+        "esbenp.prettier-vscode",
+        "ms-vsliveshare.vsliveshare"
```

Comment:
Do we need this for the project?

Reply:
Removed. I though i'd be useful if we want to dev together in the future :)